pkg/instancetype/instancetype_test.go

```diff
@@ -155,6 +187,23 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*f).To(Equal(clusterInstancetype.Spec))
 			})
 
+			It("uses client when instancetype not found within informer", func() {
+				err := clusterInstancetypeInformerStore.Delete(clusterInstancetype)
+				Expect(err).ToNot(HaveOccurred())
+
+				f, err := instancetypeMethods.FindInstancetypeSpec(vm)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(*f).To(Equal(clusterInstancetype.Spec))
+			})
+
+			It("returns expected instancetype using only the client", func() {
+				instancetypeMethods = instancetype.NewMethods(nil, nil, nil, nil, nil, virtClient)
+
+				f, err := instancetypeMethods.FindInstancetypeSpec(vm)
```

Comment:
Can you use the same naming scheme for the newly added tests as below and create a follow up PR to cleanup the existing ones?

Reply:
ACK will address in a follow up PR.

Reply:
https://github.com/kubevirt/kubevirt/pull/9067